week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/src/framework/dom-creators.ts

```diff
@@ -1,12 +1,12 @@
-function createElement(tagName = 'div', className = '') {
-  const element = document.createElement(tagName)
+function createElement(tagName = 'div', className = ''): HTMLElement {
```

Comment:
Ten `:HTMLElement` nas tutaj troszkę ogranicza bo to nie zawsze będzie to - będzie to zależne od `tagName` co zostanie przygotowane.